build.gradle

```diff
@@ -55,19 +72,15 @@ javadoc {
 	 * building the client javadocs. If we ever need the full javadocs make a new task for the
 	 * client java docs
 	 */
-	/* TODO DOCS currently java-common and auth links don't work. ROI for fixing them is low.
-	 *           Ideally, in the future make github actions publish docs on release and ref them
-	 *           here.
-	 *           https://github.com/marketplace/actions/deploy-publish-javadoc
-	 *           https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-gradle-registry
-	 */
 	/* TODO DOCS the current sdk documenation looks like invalid html to javadoc
```

Comment:
fingers crossed this will be removed soon anyway, but there's a T missing in documentation

Reply:
Fixed